### PR TITLE
linting added

### PR DESCRIPTION
**Pull request by matt-oc**
#10

### CODE REVIEW

**Comment by matt-oc** [comment]
@mcdonnelldean @Georgette

File: npm-update.js [hunk 1]
```diff
@@ -1,152 +1,145 @@
-/* Copyright (c) 2014-2015 Richard Rodger, MIT License */
-/* jshint node:true, asi:true, eqnull:true */
-"use strict";
+'use strict'
 
 
-var follow = require('follow');
-var npm_stats = require('npm-stats')
-var async     = require('async')
+var FOLLOW = require('follow')
```
Comment (mcdonnelldean) [inline npm-update.js]: Caps not CAPS

**Comment by matt-oc** [comment]
Fixed now, Could have sworn we used to use all caps instead of just first letter, could be old age though 😄

**Comment by mcdonnelldean** [comment]
@matt-oc Sooo, this is going to break once I accept @Georgette's work :D

**Comment by Georgette** [comment]
@matt-oc apologies -- you set up and linted the repo like a boss, but unfortunately all the code changed underneath you. I'm working on testing now, so I took care of lint :) you are the best!

**Comment by matt-oc** [comment]
Np didn't realise you were working on it at the same time :)

**Pull request closed without merging**